include/d/actor/d_a_player_main.h

```diff
@@ -43,6 +43,10 @@ class daPy_sightPacket_c : public dDlst_base_c {
     void setSightTex(void* sightTex) { mpSightTex = sightTex; }
     void setLockTex(void* lockTex) { mpLockTex = lockTex; }
     void setImage(ResTIMG* image) { mpImg = image; }
+    u8 checkSEFrame() { return field_0x6; }
+    void incFrame() { field_0x6++; }
+    // This inline isn't in the map but I couldn't think of another clean way to do it
+    void resetFrame() { field_0x6 = 0; }
```

Comment:
most likely some of the other inlines are supposed to be doing multiple things at once. some of the ones above I implemented might be incomplete since I filled them in based on name, I'll take a closer look later.

Reply:
Took the guess that a bit more was inside `incFrame()`, there's only 1 other time the value gets incremented (in `setHookshotSight()`) and it seems to follow the pattern in the new implementation

Reply:
for future reference, by checking the exact size in bytes of the inlines in the debug maps, I'm pretty sure these two inlines are supposed to be like this:
```cpp
void onLockFlg() {
    mLockFlag = true;
    mFrame = 0;
}
void incFrame() {
    mFrame++;
    if(mFrame == 26) {
        mFrame = 0;
    }
}
```